config/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "base_url" {
+  type        = string
+  description = "The base url for the keycloak server to target"
+  sensitive   = true
+}
+
+variable "tf_client_id" {
```

Comment:
Why are the variable names prefixed tf_? This follow the confusion over what the client resource is for. If it's for the terraform client for the GitHub flow then these values wouldn't be passed in as variables. the client would be created manually and the state would be imported into the resources defined in Terraform

Reply:
I think we need a provider set up to be able to import clients from GitHub? These variable are assigned from secrets on Github, when the GitHub flow runs, so shouldn't be visible if that was the issue?

Reply:
The variables are named tf_ to indicate they are variables for the terraform client, is this not the preferred naming convention?